### ISSUE TITLE
IsBool should handle v:true and v:false

### ISSUE THREAD

**Issue by flwyd** [google/vim-maktaba#240]
`maktaba#ensure#IsBool(v:true)` currently throws a WrongType exception because the type of `v:true` and `v:false` is boolean rather than number.  It seems to me that boolean values should be handled as well.

Additionally, the absence of `maktaba#value#IsBool` seems like a notable omission.

**Comment by dbarnett** [comment]
Fair point, might as well accommodate those (and then bump maktaba's version so you could reliably depend on it).
